projects/proposals/z:OSMF Workflows.md

```diff
@@ -0,0 +1,60 @@
+* Name of project (must be unique within OMP)
+z/OSMF Workflows
+* Project description (what it does, why it is valuable, origin and history)
+
+This projects enables customers to share z/OSMF workflows. z/OSMF workflows modernizes the system management tasks performed day to day by z/OS System Programmers. Sharing of z/OSMF workflows for system management will help customers to learn from each other.
+
+* Statement on alignment with OMP charter mission
+
+OMP is for mainframe cooperation and it's helpful to fulfill our mission to collaborate on z/OSMF workflows development.
+
+* Sponsor from TSC (sponsor helps mentor projects)
+
+John Mertic
+
+* Preferred maturity level (see [project stages](../../process/project_stages.md))
+
+Not still in incubate state but workflow technology is very mature.
+
+* License and contribution guidelines (refer to the [OMP guidelines](contribution_guidelines.md))
+
+No License.
+
+* Source control (GitHub by default)
+
+Github
+
+* External dependencies (including licenses)
+
+None
+
+* Initial committers (how long working on project)
+
+IBM, Rabobank
+
+* Infrastructure requests (CI / OMP Cluster)
+
+TBD. This is source code only and may not need any CI.
+
+* Communication channels (slack, irc, mailing lists)
+
+Currently no channels but would prefer synergy with OMP and use of Slack
+
+* Issue tracker (GitHub by default)
+
+Github
+
+* Website
+
+
+* Release methodology and mechanics
+
+Commit and push to Github
+
+* Social media accounts
+
+Does not exist yet.
+
+* Community size and any existing sponsorship
+
+Rabobank would be initial sponsors. But it will grow as more customer learn about this.
```

Comment:
Great to hear on Rabobank - would love to have them engaged and leading the work.

Reply:
Once we have community setup, I expect they will provide leadership. I don't know yet if they would offer some one from their team to be on TSC.

Reply:
Excellent!